scraper.conf

```diff
@@ -4,11 +4,12 @@ SomeString = Hello World!
 SomeInteger = 10 # an inline comment
 SomeFloat = 20.05
 SomeBoolean = true
-SomeArray = { 1, 2, 3 } # Potentially use this when I start doing multiple years
+YearsToScrape = { 2022 } # Potentially use this when I start doing multiple years
 YearToScrape = 2022
 
 
 [Pages]
+UrlPattern = "https://www.nflmockdraftdatabase.com/big-boards/{year}/consensus-big-board-{year}"
```

Comment:
Just so I understand, are the existing URL values like 2022Url no longer being used?  Same with YearToScrape now that YearsToScrape is being used?

Reply:
Yes. The idea is that URLs like `2022Url` would be added only when the URL to be scraped for a specific year does not match the general pattern as defined in `UrlPattern`. Otherwise, we can get away without defining them.


> However, it looks like YearToScrape is no longer being used now that YearsToScrape is available?

Yes. My understanding was that `YearToScrape` was to be used when scraping data for only one year. That can still be achieved by passing a single value in `YearsToScrape`.

Reply:
👍